src/main/resources/application.yml

```diff
@@ -1,21 +1,20 @@
 # CAS configuration
-# https://apereo.github.io/cas/6.3.x/configuration/Configuration-Properties.html
+# https://apereo.github.io/cas/6.0.x/configuration/Configuration-Properties.html
```

Comment:
Tämä versio ei kai sentään laske alaspäin? Olen yrittänyt pitää tämän linkin ajan tasalla.

Reply:
joo tää oli joku mun aivotus et laitoin sen tohon et pääsin tutkimaan vanhan version propseja sit kopioimalla ton "valmiin" linkin

Reply:
ja tää pitäs olla takaisin 6.3:ssa. tosiaan se oli mun oma säätö niin korjasin sen tossa toisessa commitissa takaisin

Reply:
ja sen commitin message oli kevätsiivous